MyReadingList/MyReadingList/MyReadingListView.swift

```diff
@@ -10,21 +10,19 @@ import SwiftUI
 
 struct MyReadingListView: View {
     var body: some View {
-        VStack {
-            Text(NSLocalizedString("ui.myReadingList.title", value: "My Reading List", comment: "title for the MyReadingList screen"))
-                .font(.headline)
-            Spacer()
-            
-            List {
-                HStack{
-                    Button {
-                        //do nothing for now
+        NavigationView {
+            VStack {
+                List {
+                    HStack{
+                        NavigationLink {
+                                    SearchView()
                     } label: {
-                        Image(systemName: "plus.circle")
+                            Image(systemName: "plus.circle")
```

Comment:
You now have "plus" icon in this cell as well as an arrow on the right (run it in the simulator if you are not sure what I mean)

It seems excessive to have 2 indicators that this is a tappable element. I would choose one and get rid of the other. Up to you which one to keep.

Reply:
I will keep the default indicators the arrow to describe this is a tappable element and will remove the '+' icon.

Reply:
Sounds good